### ISSUE TITLE
Question about arguments

### ISSUE THREAD

**Issue by Ganelous04** [partofthething/joy-ptz-cam#1]
Hi,
your code is exactly what I am looking for but I have an issue using it.
I am brand new in python and I have very basics object programming knowledge so sorry if this is a stupid question.
First, I modified the credential file using cam's IP and login and password of my two cameras
When I launch the code :
`python -m joyptz --config credentials.json cam1`

I obtain the error :
```
usage: __main__.py [-h] [--config CONFIG] [--output] camname control
__main__.py: error: the following arguments are required: control
```

So I understand there is a missing argument, but when I had "joystick" in the arg (seen that in the code), I have this result :
```
py -m joyptz --config credentials.json cam1 joystick
Traceback (most recent call last):
  File "C:\Users\obs\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\obs\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "C:\python\joy-ptz-cam-main\env\lib\site-packages\joyptz-0.1-py3.10.egg\joyptz\__main__.py", line 27, in <module>
    camera = cam.Camera(config[args.camname])
  File "C:\python\joy-ptz-cam-main\env\lib\site-packages\joyptz-0.1-py3.10.egg\joyptz\cam.py", line 36, in __init__
    self.init_camera(config)
  File "C:\python\joy-ptz-cam-main\env\lib\site-packages\joyptz-0.1-py3.10.egg\joyptz\cam.py", line 41, in init_camera
    mycam = ONVIFCamera(
  File "C:\python\joy-ptz-cam-main\env\lib\site-packages\onvif_zeep-0.2.12-py3.10.egg\onvif\client.py", line 216, in __init__
    self.update_xaddrs()
  File "C:\python\joy-ptz-cam-main\env\lib\site-packages\onvif_zeep-0.2.12-py3.10.egg\onvif\client.py", line 223, in update_xaddrs
    self.devicemgmt  = self.create_devicemgmt_service()
  File "C:\python\joy-ptz-cam-main\env\lib\site-packages\onvif_zeep-0.2.12-py3.10.egg\onvif\client.py", line 333, in create_devicemgmt_service
    return self.create_onvif_service('devicemgmt', from_template)
  File "C:\python\joy-ptz-cam-main\env\lib\site-packages\onvif_zeep-0.2.12-py3.10.egg\onvif\client.py", line 312, in create_onvif_service
    xaddr, wsdl_file, binding_name = self.get_definition(name, portType)
  File "C:\python\joy-ptz-cam-main\env\lib\site-packages\onvif_zeep-0.2.12-py3.10.egg\onvif\client.py", line 292, in get_definition
    raise ONVIFError('No such file: %s' % wsdlpath)
onvif.exceptions.ONVIFError: Unknown error: No such file: C:\python\joy-ptz-cam-main\env\lib\site-packages\onvif_zeep-0.2.12-py3.10.egg\wsdl\devicemgmt.wsdl
```
Can you give examples of possible arguments?
Is this wrigth to replace "camera1.local" by the cam's IP?

Thanks for your help

**Comment by partofthething** [comment]
Sounds like you're doing it right, and yes camera1.local with cam's IP is right. I think what's happening here is that there's a missing ONVIF definition file that's part of the underlying[ onvif-zeep library](https://github.com/FalkTannhaeuser/python-onvif-zeep) that my code uses as a dependency. 

The missing file is the one in [here](https://github.com/FalkTannhaeuser/python-onvif-zeep/tree/zeep/wsdl) but I'm not sure why your system isn't finding it. It should have been installed when you installed the library. You may have to adjust the directory it's looking in based on where those files got installed to. 

Similar to https://github.com/FalkTannhaeuser/python-onvif-zeep/issues/13

**Comment by Ganelous04** [comment]
Hi,
thanks for your quick answer!
Found the problem with your clue : the problem was the address of the file. It is located in [...]onvif_zeep-0.2.12-py3.10.egg\Lib\site-packages\wsdl
I corrected the "def __init__" line and it starts.

But with an other error about axes-vals... I am looking for an explanation and will open another question if necessary
(Just for keeping me searching for nothing : I want to use a gamepad - with 2 joysticks - not a 'simple' joystick, is the script theoricaly compatible?)
Thanks
